packages/sonarwhal/src/lib/cli/rules/new-rule.ts

```diff
@@ -9,15 +9,14 @@ import { CLIOptions } from '../../types';
 import * as logger from '../../utils/logging';
 import { writeFileAsync } from '../../utils/misc';
 import {
-    processDir, packageDir, questions, normalize,
+    processDir, packageDir, questions, normalize, // eslint-disable-line no-unused-vars
```

Comment:
`eslint-disable-line` shouldn't be necessary. Why do you need it?

Reply:
@sarvaje `processDir` keeps getting reported of not used, but it's used in `path.join` later on.